src/visualizer.jl

```diff
@@ -247,6 +250,16 @@ function setanimation!(vis::Visualizer, anim::Animation; play::Bool=true, repeti
     send(vis.core, cmd)
 end
 
+"""
+Take a screenshot of the current visualizer.
+
+$(TYPEDSIGNATURES)
+"""
+function takescreenshot!(vis::Visualizer)
```

Comment:
Perhaps `save_image` as well, to match https://github.com/rdeits/meshcat/pull/79#discussion_r563339743 ?